<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color:rgb(238,238,238)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_File_Name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>19</y>
      <width>1041</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-color: 1px solid rgb(142,142,142)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>FILE NAME</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_File_Duration">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>881</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider:groove:horizontal
{
	height: 10px;
	width: 850px;
	background: darkgray;
	border-radius:5px;
}
QSlider:handle:horizontal
{
	background:rgb(90,90,90);
	width: 20px;
	height: 20px;
	border-radius:3px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_Volume">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider:groove:horizontal
{
	height: 10px;
	width: 350px;
	background: darkgray;
	border-radius:5px;
}
QSlider:handle:horizontal
{
	background:rgb(90,90,90);
	width: 20px;
	height: 20px;
	border-radius:3px;
	margin: -7 7;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:none;
	border-radius:25px;
	background-color:darkgray;
	color:rgb(252,252,252);
	padding:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:rgb(0,0,0)
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:rgb(0,0,0)
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>501</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Back">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:33px;
	background-color:darkgray;
	color:rgb(252,252,252);
	min-width:66px;
	max-width:66px;
	min-height:66px;
	max-height:66px;
	padding:0px;
}

QPushButton:Hover {
	background-color:rgb(210,210,210);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:33px;
	background-color:darkgray;
	color:white;
	min-width:66px;
	max-width:66px;
	min-height:66px;
	max-height:66px;
	padding:0px;
}
QPushButton:Hover {
	background-color:rgb(210,210,210);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Play">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:33px;
	background-color:darkgray;
	color:rgb(252,252,252);
	min-width:66px;
	max-width:66px;
	min-height:66px;
	max-height:66px;
	padding:0px;
}
QPushButton::icon {
    color:rgb(252,252,252) ;
}
QPushButton:Hover {
	background-color:rgb(210,210,210);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Pause">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:33px;
	background-color:darkgray;
	color:rgb(252,252,252);
	min-width:66px;
	max-width:66px;
	min-height:66px;
	max-height:66px;
	padding:0px;
}
QPushButton:Hover {
	background-color:rgb(210,210,210);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Forward">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:33px;
	background-color:darkgray;
	color:rgb(252,252,252);
	min-width:66px;
	max-width:66px;
	min-height:66px;
	max-height:66px;
	padding:0px;
}
QPushButton:Hover {
	background-color:rgb(210,210,210);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_Audio_File"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <action name="actionOpen_Audio_File">
   <property name="text">
    <string>Open Audio File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
